frontend/packages/dev-console/src/components/pipelines/pipeline-builder/task-sidebar/temp-utils.tsx

```diff
@@ -87,6 +87,7 @@ export const ArrayParam: React.FC<ParameterProps> = (props) => {
         {description}
       </p>
       <MultiColumnFieldFooter
+        addLabel="Add another value"
```

Comment:
how about `add value` ?

Reply:
Same thing here, the [value before it was removed](https://github.com/openshift/console/pull/5998/files#diff-6133e47a6edec3216bd17358565eef74L88).